LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs

```diff
@@ -529,6 +529,52 @@ private static void GetBoardSpecificConfiguration
 
                         break;
 
+                    case Manufacturer.MSI when model == Model.Z790_GODLIKE_MAX: // NCT6687D
+                        v.Add(new Voltage("+12V", 0));
+                        v.Add(new Voltage("+5V", 1));
+                        v.Add(new Voltage("Vcore", 2));
+                        v.Add(new Voltage("Voltage #1", 3));
+                        v.Add(new Voltage("DIMM", 4));
+                        v.Add(new Voltage("CPU I/O", 5));
+                        v.Add(new Voltage("CPU System Agent", 6));
+                        v.Add(new Voltage("Voltage #2", 7));
+                        v.Add(new Voltage("AVCC3", 8));
+                        v.Add(new Voltage("CPU Termination", 9));
+                        v.Add(new Voltage("VRef", 10));
+                        v.Add(new Voltage("VSB", 11));
+                        v.Add(new Voltage("AVSB", 12));
+                        v.Add(new Voltage("CMOS Battery", 13));
+
+                        t.Add(new Temperature("CPU", 0));
+                        t.Add(new Temperature("System", 1));
+                        t.Add(new Temperature("VRM MOS", 2));
+                        t.Add(new Temperature("PCH", 3));
+                        t.Add(new Temperature("CPU Socket", 4));
+                        t.Add(new Temperature("PCIE_1", 7));
+                        t.Add(new Temperature("PCIE_2", 8));
+                        t.Add(new Temperature("M2_1", 9));
+                        t.Add(new Temperature("M2_4", 10));
```

Comment:
_X are their numbers I'm guessing?

```suggestion
                        t.Add(new Temperature("PCIE #1", 7));
                        t.Add(new Temperature("PCIE #2", 8));
                        t.Add(new Temperature("M2 #1", 9));
                        t.Add(new Temperature("M2 #4", 10));
```